R/cookies.R

```diff
@@ -246,3 +247,43 @@ cookie_banner_server <- function(
     }))
   })
 }
+
+#' init_cookies
+#'
+#' @description
+#' init_cookies() creates a local copy of the JavaScript file required for cookies to work.
+#' It checks whether there is already a www/ folder and if not, it creates one
+#' It then checks whether the cookie-consent.js file exists in the www/ folder
+#' If the file exists, it will print a message in the console to let you know
+#' If the file doesn't exist, it will pull a copy from the GitHub repo
+#' If it cannot connect to the repo then it will print "Download failed".
+#' No input parameters are required
+#'
+#' Call init_cookies() in the console to run the function
+#'
+
+
```

Comment:
init_cookies currently isn't exported as a user function, so won't work when people load the package. Need to add the @export line to the roxygen comments (can check the other functions for how it needs to be done).